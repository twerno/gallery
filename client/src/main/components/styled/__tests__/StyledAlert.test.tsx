import * as React from 'react';
import { componentBasicTests } from "test/componentBasicTests";
import StyledAlert from "../StyledAlert";

describe("StyledAlert", () => {

    componentBasicTests("StyledAlert", <StyledAlert />);
});