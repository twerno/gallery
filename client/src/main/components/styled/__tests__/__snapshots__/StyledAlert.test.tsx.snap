// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledAlert [StyledAlert] Component basic tests matches the snapshot 1`] = `
.c0 {
  padding: 9px 18px;
  width: 100%;
  font-size: 1.5rem;
  color: red;
}

<div
  className="c0"
/>
`;
